### DISCUSSION TITLE
Github team users vs collaborators pricing.

### DISCUSSION THREAD

**Discussion by paulwizviz** [community/community#22012] [New to GitHub]
<p>Hi</p>
<p>     I understand with the latest pricing, Github Team is free or $4 per user/month (paid).</p>
<p>    For the freemium version, I presumed you can add as many users as you like and there is no charge. Is that right?</p>
<p>    If so how does Github Team disambiguate between freemium and paid version? </p>
<p>    If for example, my freemium account had 10 users does it mean when upgrading to a paid account, I have to pay for all 10 users?</p>
<p>     Also when adding a collaborator, my I also paid for the collaborator? </p>

**Comment by ghost** [comment, marked as answer]
<p>Hello <a class="mention" href="/paulwizviz">@paulwizviz</a>,</p>
<p>You’re correct; GitHub Free allows you to add as many members and collaborators to your organization as you’d like at no charge.</p>
<p>If you decide to upgrade to GitHub Team, however, you must <a href="https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/upgrading-your-github-subscription#upgrading-your-organizations-subscription" rel="nofollow noopener">add a paid seat for each organization member as well as each collaborator</a>. In your example, upgrading an 10-user organization from GitHub Free to GitHub Team would require 10 paid seats at $4 per month each, for a total charge of $40 per month.</p>

**Comment by paulwizviz** [comment]
<p>Could you elaborate on charges for collaborators?</p>
<p>I understand they are being treated differently from users.</p>
<p>Do I have to pay for every collaborators I add to GitHub Team - I.e. paid version?</p>

**Comment by ghost** [comment]
<p>Although <a href="https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/adding-outside-collaborators-to-repositories-in-your-organization" rel="nofollow noopener">collaborators and organization members serve different purposes within GitHub</a>, for organizations subscribed to GitHub Team collaborators must occupy a paid seat in order to access private repositories. Organization members always require a paid seat.</p>

**Comment by tanetane2** [comment]
<p>Hey,<br>
I have a similar question regarding charging per seat.<br>
Basically, can you explain to me how do you handle it when i add a new user to the organization?<br>
For example: i have 10 users x $4 = $40.<br>
For example, I add a new user on the 20th of March. How do you handle it and when you will charge me for a new user.<br>
If I am right, you will charge one user licence(4$) but for the remaining days I March only, not a full month.<br>
But what if the month has 28 days, or 30, or 31.</p>
<p>Thanks in advance.</p>

**Comment by laurie** [comment]
<p>Hi there,</p>
<p>If you have an existing subscription to GitHub Team with 10 users, then add another seat during the billing cycle, you’ll be charged a prorated amount of $4 to cover the period between when you added the seat and when the subscription typically renews.</p>
<p>As an example, imagine you pay the $40 for your subscription on the 5th monthly, then add a seat on the 20th, you’ll be charged ~$2 to cover that seat through to the renewal date, then $44 on the 5th monthly going forward.</p>
<p>I hope that helps!</p>

**Comment by interglobalmedia** [comment]
Hi,

Is there any way of getting users who want to be part of your team to pay the 4 dollars a month? Instead of the organization paying for it? I am an organization of 1, and my team members would be students of an online course I am creating. Is there a way of getting them to pay the 4 dollars a month while they are actively members of the team associated with my organization's repository? Thanks!

**Comment by interglobalmedia** [comment]
I have another question. The base role that is associated with my organization's repository for team members (via my pro plan), since it is read only, I hope that means that those team members cannot modify anything on that repository? Because that iks what I want. Thanks! Then there would be no problem regarding keeping my current set up (pro plan). Thanks!
